databricks/koalas/tests/test_stats.py

```diff
@@ -26,149 +24,150 @@
+                                'f': np.array([1.0, 2.0, 3.0]),
+                                'g': np.array([True, False, True]),
+                                'h': np.array(list('abc'))},
+                               index=pd.Index([1, 2, 3], name='myindex'))
+            kdf = koalas.from_pandas(pdf)
+            self.assert_eq(kdf.corr(), pdf.corr())
 
     def test_stats_on_boolean_dataframe(self):
-        df = pd.DataFrame({'A': [True, False, True],
-                           'B': [False, False, True]})
-        ddf = koalas.from_pandas(df)
+        pdf = pd.DataFrame({'A': [True, False, True],
+                            'B': [False, False, True]})
+        kdf = koalas.from_pandas(pdf)
 
-        pd.testing.assert_series_equal(ddf.min(), df.min())
-        pd.testing.assert_series_equal(ddf.max(), df.max())
+        pd.testing.assert_series_equal(kdf.min(), pdf.min())
+        pd.testing.assert_series_equal(kdf.max(), pdf.max())
 
-        pd.testing.assert_series_equal(ddf.sum(), df.sum())
-        pd.testing.assert_series_equal(ddf.mean(), df.mean())
+        pd.testing.assert_series_equal(kdf.sum(), pdf.sum())
+        pd.testing.assert_series_equal(kdf.mean(), pdf.mean())
 
-        pd.testing.assert_series_equal(ddf.var(), df.var())
-        pd.testing.assert_series_equal(ddf.std(), df.std())
+        pd.testing.assert_series_equal(kdf.var(), pdf.var())
+        pd.testing.assert_series_equal(kdf.std(), pdf.std())
 
     def test_stats_on_boolean_series(self):
-        s = pd.Series([True, False, True])
-        ds = koalas.from_pandas(s)
+        ps = pd.Series([True, False, True])
+        ks = koalas.from_pandas(ps)
 
-        self.assertEqual(ds.min(), s.min())
-        self.assertEqual(ds.max(), s.max())
+        self.assertEqual(ks.min(), ps.min())
+        self.assertEqual(ks.max(), ps.max())
 
-        self.assertEqual(ds.sum(), s.sum())
-        self.assertEqual(ds.mean(), s.mean())
+        self.assertEqual(ks.sum(), ps.sum())
+        self.assertEqual(ks.mean(), ps.mean())
 
-        self.assertAlmostEqual(ds.var(), s.var())
-        self.assertAlmostEqual(ds.std(), s.std())
+        self.assertAlmostEqual(ks.var(), ps.var())
+        self.assertAlmostEqual(ks.std(), ps.std())
 
     def test_some_stats_functions_should_discard_non_numeric_columns_by_default(self):
-        df = pd.DataFrame({'i': [0, 1, 2],
-                           'b': [False, False, True],
-                           's': ['x', 'y', 'z']})
-        ddf = koalas.from_pandas(df)
+        pdf = pd.DataFrame({'i': [0, 1, 2],
+                            'b': [False, False, True],
+                            's': ['x', 'y', 'z']})
+        kdf = koalas.from_pandas(pdf)
 
         # min and max do not discard non-numeric columns by default
-        self.assertEqual(len(ddf.min()), 3)
-        self.assertEqual(len(ddf.max()), 3)
+        self.assertEqual(len(kdf.min()), len(kdf.min()))
```

Comment:
Starting from here to the end of the file, you keep comparing `kdf` with `kdf`. That's probably some copy & paste error but it should, of course, compare `kdf` with `pdf`.

Reply:
Right, not very useful ;-)